necroHeal.lic

```diff
@@ -0,0 +1,43 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placement comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status

```

Comment:
Right now `;safe-room` checks if you are a necro and quits. That check could run this healing logic instead of exiting with the added benefit of being integrated into `;training-manager`. Sounds like a win-win